config/scheduling.php

```diff
@@ -10,7 +10,7 @@
 
     'collection_class' => [
 
-        'slot' => 'SlotsCollection'
+        'slot' => 'SlotsCollection' 
```

Comment:
@drewroberts I don't know what exactly does this collection_class

Reply:
It includes slots and our virtual slots. @sl0wik can provide more details

Reply:
ok, I saw its required in the slot model, so it will be moved to booking package as well